src/Common/src/System/Dynamic/Utils/DelegateHelpers.cs

```diff
@@ -10,6 +10,10 @@
 using System.Runtime.CompilerServices;
 using System.Reflection;
 
+#if FEATURE_REFEMIT
```

Comment:
The flag is set by special project that produces debug versions of System.Linq.Expressions and System.Dynamic.Runtime for NetNative. This part is likely to change once project files are redirected to use these sources.